chapters/annotations.tex

```diff
@@ -1963,14 +1971,14 @@ \subsection{Version Date and Build Information}\label{version-date-and-build-inf
 
 Besides version information, a top-level class can have additionally the following top-level annotations to specify associated information to the version number:%
 \begin{lstlisting}[language=modelica]
-String versionDate   "UTC date of first version build (in format: YYYY-MM-DD)";
-Integer versionBuild "Larger number is a more recent maintenance update";
-String dateModified  "UTC date and time of the latest change to the package
-                      in the following format (with one space between date
-                      and time): YYYY-MM-DD hh:mm:ssZ";
-String revisionId    "Revision identifier of the version management system used
-                      to manage this library. It marks the latest submitted
-                      change to any file belonging to the package";
+/*literal*/ constant String versionDate   "UTC date of first version build (in format: YYYY-MM-DD)";
+/*literal*/ constant Integer versionBuild "Larger number is a more recent maintenance update";
+/*literal*/ constant String dateModified  "UTC date and time of the latest change to the package
```

Comment:
Considering the line breaks we had since before, I would guess that we now exceed the page width.  Perhaps better to reformat with the description string starting on the line below?